Indexer-Spikes/Substreams-Soroban-Events/src/mapping.ts

```
@@ -0,0 +1,37 @@
+import { BigInt, Bytes, log } from "@graphprotocol/graph-ts";
```

Biome (2.1.2)

[error] Do not shadow the global "BigInt" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)